styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 30px;
+    font-family: 'Montserrat', sans-serif;
+    color: black;  
+}
+
+
+.page {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    background-image: url(../img/QuizBackground.png);
+    background-size: cover;
+    padding: 3%;
+}
+
+.menu{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 2%;
+}
+
+.menu__logo {
+    width: 25%;
+}
+
+.menu__list {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 65%;
+    height: 110px;
+    background-color: white;
+    border-radius: 16px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+}
+
+.menu__item {
+    position: relative;
+    padding: 10px;
+    width: 20%;
```

Comment:
zamiast dawać width: 20% ja zazwyczaj używam do tego paddingu, tak samo wyżej w 39 linijce height: 110px można zrobić paddingiem typu 1rem góra dół, jest responsywne zamiast na sztywno pixelami.

Reply:
Nie umiem inaczej budować menu, tylko z użyciem width dla poszczególnych komórek, wtedy wszystko fajnie i równo się rozkłada. Ale jak masz inny pomysł i ochotę to śmiało, możesz edytować kod :)